xpasses/XS002/testdata/src/a/alias.go

```diff
@@ -0,0 +1,22 @@
+package a
+
+import (
+	s "github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```

Comment:
If you look on the main branch, you should now see a bunch of `_v2.go` files to cover Terraform Plugin SDK v2. Can you please create a matching v2 file for this please? For schema testing, usually it just involves copy-pasting the v1 file and changing this line in the new file, e.g.

```go
s "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```